packages/electron-app/src/main/util/substrateProcess.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { spawn } from 'child_process';
+import Pino from 'pino';
+
+const pino = Pino();
+
+import { bundledPath } from './staticPath';
+
+// TEMPORARY: change to runSubstrateLight once the light client is available.
+export const runSubstrateDev = (setPid?: (pid: number) => void) => {
+  pino.info('running substrate dev ....');
+  const substrate = spawn(bundledPath, ['--dev']); // FIXME: --light
+
+  substrate.stdout.on('data', data => pino.info(data.toString()));
+  substrate.stderr.on('data', error => {
+    pino.error(error.toString());
+  });
+  substrate.on('error', error => {
+    pino.error('Substrate process errored => ', error.toString());
+    purgeDevChain();
```

Comment:
I'm thinking: should we actually do this? Esp if the user has already done some stuff on his chain

Reply:
I'd say it is okay on light client, assuming it is able to sync relatively fast. They won't be authoring blocks anyway so I don't think any information would be lost by doing this, only time.